### PR TITLE
chore: update dependenices

### PR DESCRIPTION
**Pull request by JasonLantz**
update webpack,elements, scripts, prop-types,webpack-dev-server,babel-loader

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,8 +77,9 @@
     "style-loader": "^0.13.1",
     "url-loader": "^0.5.7",
     "uuid": "2.0.2",
-    "webpack": "^2.2.1",
-    "webpack-dev-server": "^2.3.0"
+    "webpack": "^3.4.1",
```
Comment (wyseguyonline) [inline package.json]: Can we verify that the new version of webpack works with the other dependencies?  IIRC, there were a couple modules that also needed to be upgraded.

Comment (JasonLantz) [inline package.json]: i added any unmet dependencies.... babel loader was one.... prop-types was the other.  Then started up the demo.  i'm not sure of the other dependencies... i'll look at the others, style loader is up to 0.18.... and i'm not sure we need fs-extra

Comment (StommePoes) [inline package.json]: wasn't fs-extra for the shell scripts that loaded icons/fonts from elements? If so then I agree, we prolly don't need it. (and \o/ less worry about Windows-with-only-powershell-which-has-no-&&-for-some-stupid-reason)

Comment (JasonLantz) [inline package.json]: i believe that to be the case....

**Pull request closed without merging**